package.json

```diff
@@ -25,6 +25,7 @@
     "devDependencies": {
         "@definitelytyped/dtslint-runner": "latest",
         "danger": "^10.1.1",
+        "dts-gen": "^0.6.0",
```

Comment:
Don't add dependencies to the root `package.json`, thanks

Reply:
Done!